projects/storefrontlib/src/cms-components/asm/loader/asm-loader.component.html

```diff
@@ -0,0 +1,8 @@
+<!-- 
+    We defer loading of the real AMS UI, so that once we have lazy loading in place,
+    the UI will only be loaded for ASM users.
+-->
```

Comment:
html comments are removed by angular template processing, right?

Reply:
yes, in production build it is.